components/Modal/CreateBlueprint.js

```diff
@@ -105,26 +108,45 @@ class CreateBlueprintModal extends React.Component {
   }
 
   dismissErrors() {
-    this.setState({ errorNameEmpty: false, errorNameDuplicate: false, errorInline: false });
+    this.setState({
+      errorNameEmpty: false,
+      errorNameDuplicate: false,
+      errorNameSpace: false,
+      errorInline: false,
```

Comment:
This has no reason to be in the state. The function you introduced nameContainsError can return it's value to a normal variable in the render function and the result can be reused there.
But since you didn't introduce this, you don't have to change it now.